aws/backbone/variables.tf

```diff
@@ -41,3 +41,15 @@ variable "tags" {
 variable "vpc_cidr" {
   description = "VPC CIDR block"
 }
+
+variable "vgw_id" {
+  description = "ID of a VGW to attach to the VPC. No attachment if empty"
```

Comment:
Can we detail `VGW` for Virtual Gateway, putting the acronym between parenthesis? as in: `ID of a Virtual Gateway (VGW) to attach to the VPC...`

Reply:
Yes, that would be easier to read